packages/server-core/src/media/storageprovider/ipfsstorageprovider.interface.ts

```diff
@@ -0,0 +1,122 @@
+import { FileContentType } from '@xrengine/common/src/interfaces/FileContentType'
+
+export interface IPFSStorageObjectInterface {
+  Key?: string
+  Body: Buffer
+  ContentType?: string
+}
+
+export interface IPFSStorageListObjectInterface {
+  Prefix?: string
+  IsTruncated?: boolean
+  NextContinuationToken?: string
+  Contents: { Key: string }[]
+  CommonPrefixes?: { Prefix: string }[]
+}
+
+export interface SignedURLResponse {
+  fields: {
+    [key: string]: string
+  }
+  url: string
+  local: boolean
+  cacheDomain: string
+}
+
+export interface BlobStore {
+  path: string
+  cache: any
+  createWriteStream(options: string | { key: string }, cb?: (err, result) => void)
+  createReadStream(key: string | { key: string }, options?: any)
+  exists(options: string | { key: string }, cb?: (err, result) => void)
+  remove(options: string | { key: string }, cb?: (err, result) => void)
+}
+
+export interface IPFSStorageProviderInterface {
```

Comment:
lets get rid of this interface, and just use StorageProviderInterface and implement the unused functions as no-ops